app/views/widgets/donation/_display.slim

```diff
@@ -0,0 +1 @@
+#donation
```

Comment:
Missing newline (obviously, this is just a stub right now, but if it's going into dev we should try to make it meet standards)